lua/keybinds.lua

```diff
@@ -1,4 +1,4 @@
-vim.api.nvim_set_keymap("n", "<C-Q>", ":BufferClose<CR>", { noremap = true })
+vim.api.nvim_set_keymap("n", "<C-Q>", ":bd<CR>", { noremap = true }) -- BufferClose isnt the correct command to close a buffer ;)
```

Comment:
Can you please remove that comment :peep:, other than that, its fine!

Reply:
you can remove it urself :ugundu: